React/src/Tooltip.tsx

```diff
@@ -0,0 +1,46 @@
+import { ClickEvent } from 'devextreme/ui/button';
+import { Button } from 'devextreme-react';
+import React, { useCallback } from 'react';
+
+function Tooltip(props: any): JSX.Element {
```

Comment:
Can we add a type for ```props``` here?

Reply:
Added custom interface for this type: 
`interface TooltipProps extends AppointmentTooltipTemplateData {
  isDeleteButtonExist: boolean;

  onDeleteButtonClick: Function;

  color: string | undefined;
}`